docs/synapse/userguides/storm_ref_automation.rstorm

```diff
@@ -154,185 +180,407 @@ Triggers and Cron
+  and cron jobs. These permissions must be explicitly granted to users and/or roles. See the `Cortex permissions`_
+  section of the `Synapse Admin Guide`_ for a list of ``cron.*`` and ``trigger.*`` permissions.
+    
+- **Execution.** Both triggers and cron jobs execute with the permissions of a designated user associated
+  with the individual trigger or cron job. By default, this is the user who creates the trigger or cron
+  job. The user can be changed (e.g., for organizations that wish to use a dedicated account for
+  automation tasks) using the ``set()`` method of the :ref:`stormprims-trigger-f527` or :ref:`stormprims-cronjob-f527`
+  primitives, respectively.
   
-- **Execution.** Both triggers and cron jobs execute with the permissions **of the user who creates them.**
-  A trigger or cron job can only perform actions that their creator has permissions to perform.
+- **Scope.** Both triggers and cron jobs run **within a specific view**; any changes made to Synapse's data
+  by the trigger or cron job are made (written) to the topmost (writeable) layer of the view. This view-specific
+  behavior is transparent when using a simple Synapse implementation consisting of a single Cortex with a single
+  layer and a single view (Synapse's default configuration). Organizations using multiple views or that
+  frequently fork views should consider the impact of your view architecture on automation deployment
+  and behavior.
+
+.. NOTE::
   
-- **Introspection.** Triggers and cron jobs are created as runtime nodes ("runt nodes") in Synapse
-  (``syn:trigger`` and ``syn:cron`` nodes, respectively).
+  The use of sensitive information (e.g., credentials, API keys, etc.) in the Storm that is executed by
+  a trigger or cron job is **strongly discouraged**. If your automation needs to make use of any sensitive
+  information, we recommend creating a `Power-Up`_ with associated Storm commands. The Power-Up architecture
+  allows users to run commands that make use of sensitive information while ensuring that information is
+  not exposed. See the `Rapid Power-Up Development`_ section of the `Synapse Developer Guide`_ for details.
+
 
-- **Scope.** Both triggers and cron jobs run **within a specific view.** Synapse allows the optional
-  segregation of data in a Cortex into multiple layers (:ref:`gloss-layer`) that can be "stacked" to
-  provide a unified :ref:`gloss-view` of data to users. You must specify the particular view in which
-  each trigger or cron job runs.
+.. _auto-cron:
+
+Cron
+++++
+
+Cron in Synapse is similar to the well-known cron utility. Cron jobs execute their associated Storm
+on a specified schedule. Jobs can be written to execute once (using the :ref:`storm-cron-at` command)
+or on a recurring basis (using :ref:`storm-cron-add`).
+
+.. TIP::
   
-  .. NOTE::
+  When scheduling cron jobs, Synapse interprets all times as UTC.
+
+Cron jobs have the following characteristics in Synapse 2.xx:
```

Comment:
Seems like this should be 2.x or 2.x.x

Reply:
We probably do not need the version specific language at all.

Reply:
Removed